CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- checks if the paginationClone object is not empty in `Table`.
```

Comment:
Sounds to me that the sections should be `Changed` and `Removed`. What do you think?

Reply:
maybe in `remove propTypes required in SwitchablePagination.` better use the `Changed`. But for `checks if the paginationClone object is not empty in Table.` makes sense to use `Fixed`